content/blog/2025/06/2025-06-11-announcing-premium-tiers.md

```diff
@@ -0,0 +1,63 @@
+
+We are limited in scope and design by the fact we need to ship a minimum viable product as soon as possible (we need to reduce costs now) and by not wanting to impose too much development (if any) to Matrix client developers.
+
+Obviously we would have preferred to keep everything free of charge. We will never sell our users’data or cripple  our services with ads, so we need to find ethical sources of revenue. 
+
+### When will the new plans take effect?
+
+The roll-out will happen progressively, starting in the coming weeks and hopefully completing in the summer of 2025. We will start by opening up premium plans to new users only, before progressively migrating all existing accounts to a free plan which will give them the option to upgrade to a premium plan. Users will of course be notified ahead of  their account being migrated.
+
+### Will this work with whatever Matrix client I use? 
+
+Yes. The plan management will be handled via the [My Account](https://account.matrix.org/account/) screens provided by the Matrix Authentication Service (MAS), and notifications to users will be sent in a dedicated room using the [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices) feature built in to the Matrix protocol – already used by the homeserver to send automatic messages to the user – so should be seamless for every client. 
+
+### I am a Matrix client developer, do I need to do anything?
+
+Assuming that your client already supports [Server Notices](https://spec.matrix.org/v1.14/client-server-api/#server-notices), the only complication introduced by the change is for applications distributed via the Apple App Store. Apple places [restrictions](https://developer.apple.com/app-store/review/guidelines/#in-app-purchase) on how payments are implemented by iOS (et al) apps.
+
+We expect that most, if not all, apps that fall within scope would be classified as what Apple calls “[Free Stand-alone Apps](https://developer.apple.com/app-store/review/guidelines/#free-stand-alone-apps)”. Such apps do not need to use in-app purchases so long as “there is no purchasing inside the app, or calls to action for purchase outside of the app”.
+
+In order to meet these requirements we have proposed [MSC4286](https://github.com/matrix-org/matrix-spec-proposals/pull/4286) which provides a way for a homeserver (such as the matrix.org homeserver) to flag parts of messages as containing a call to action and for affected clients to be able to hide that content. Example implementations are linked in the MSC.
+
+### I am already supporting the Foundation by paying an individual membership, will I have to pay for a premium plan too?
+
+No, individual members of the Foundation will get access to the premium features at no extra cost. This benefit will be implemented as part of the process of migrating existing accounts to the free plan. 
```

Comment:
- could add link to membership page?
- should we already update the membership page to match? e.g. in this PR?

Reply:
Assuming the redesign happens in time?